assignment5.txt

```diff
@@ -0,0 +1,7 @@
+Armstrong Homework
+
+1.) Passing by value is basically making a copy of a variable and passing it to a function. Example (int variable)  Passing by reference changes the actual original value of the varialbe because it does not create a copy. Example, global variable and arrays.
```

Comment:
right - note that everything in C is passed by value. See http://stackoverflow.com/questions/12953496/what-happens-during-pass-by-reference-in-c